lib/truffle/rubygems/installing_extra_executables.rb

```diff
@@ -0,0 +1,38 @@
+module Gem
+  module InstallingExtraExecutables
```

Comment:
InstallExtraExecutables or InstallExtraExecutableHooks maybe.

Reply:
The name of the module should be a noun rather so the first option is not good. The hooks suffix is not really necessary, that's sort of an implementation detail how the module allows to install more executables. This matters very little, I'll keep the current name.

Reply:
`Installing` is not a noun either.
I think `InstallExtraExecutablesPlugin` would make sense.

Reply:
If you want it to be a noun, it should be `ExtraExecutablesInstaller`. You could make it a verb if you're doing something like the Command pattern, but that doesn't seem to be the case here.

Reply:
`ExtraExecutablesInstaller` works thanks.